docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+  db:
+    image: postgres
+    restart: always
+    volumes:
+      - "./postgres-data:/var/lib/postgresql/data"
+    ports:
+      - ${DB_PORT}:${DB_PORT}
```

Comment:
I suggest using another port rather than using one of the database. Because it is conflicting with Postgres port.